src/prelude/prim.mo

```diff
@@ -376,6 +377,84 @@ func stableMemoryStoreBlob(offset : Nat64, val :  Blob) : () =
 func stableVarQuery() : shared query () -> async {size : Nat64} =
   (prim "stableVarQuery" : () -> (shared query () -> async {size : Nat64})) () ;
 
+// stable regions
+
+// Returns the region assigned to the original stable memory interface.
+func stableMemoryRegion() : Region =
+  (prim "stableMemoryRegion" : () -> Region) ();
```

Comment:
@crusso `stable-region0-danger` shows why we either need to hide this, or do more work to avoid aliasing post upgrade.